axelor-project/src/main/resources/views/TeamTask.xml

```diff
@@ -193,6 +177,16 @@
 				<field name="doUpdateNextTasks" hidden="true"/>
 				<field name="hasDateOrFrequencyChanged" hidden="true"/>
 			</panel>
+			
+			<panel name="timeSpentPanel" title="Time Spent">
+				<field name="totalRealHrs" readonly="true" colSpan="4">
+				<viewer><![CDATA[
+                   	<span>{{record.totalRealHrs}} </span><span x-translate>hours</span>
+       			]]></viewer>
+				</field>
+				<panel-dashlet name="validatedTimesheetLinesPanel" action="action-teamtask-dashlet-validated-timeshet-lines,action-timesheet-line-attrs-hide-teamtask" title="Validated Timesheet lines" colSpan="12" canSearch="true"/>
```

Comment:
what is the use of action-timesheet-line-attrs-hide-teamtask?

Reply:
I checked the code and found that "action-timesheet-line-attrs-hide-teamtask" method is not used nowhere in the code can i remove it??

Reply:
yes